src/main/java/org/assertj/core/api/AbstractInputStreamAssert.java

```diff
@@ -298,4 +335,19 @@ public SELF hasDigest(String algorithm, String expected) {
     inputStreams.assertHasDigest(info, actual, algorithm, expected);
     return myself;
   }
+
+  private String readString(Charset charset) {
```

Comment:
I didn't add this method to InputStream because InputStreams is a collection of assertions and this is not an assertion.
I added the method to the bottom so that all public methods are on top.